ti_config_fragments/audio_display.cfg

```diff
@@ -34,7 +34,7 @@ CONFIG_DRM_CDNS_MHDP=m
 CONFIG_DRM_CDNS_DSI=n
 
 CONFIG_PHY_J721E_WIZ=y
-CONFIG_PHY_CADENCE_DP=y
+CONFIG_PHY_CADENCE_TORRENT=y
 
```

Comment:
Create a separate commit for changing the ti_config_fragments. They will never go upstream, so it's better to keep them in separate commits.